packages/datatrak-web/src/features/Survey/Components/SurveySideMenu/SurveySideMenu.tsx

```diff
@@ -62,6 +65,15 @@ const SurveyMenuItem = styled(ListItem).attrs({
   &:hover {
     background-color: ${({ theme }) => `${theme.palette.grey[400]}4d`};
   }
+  .MuiButton-label {
+    > span:not(.MuiTouchRipple-root) {
+      display: -webkit-box;
+      -webkit-box-orient: vertical;
+      line-clamp: ${({ $isInstructionOnly }) => ($isInstructionOnly ? 1 : 2)};
+      -webkit-line-clamp: ${({ $isInstructionOnly }) => ($isInstructionOnly ? 1 : 2)};
+      overflow: hidden;
```

Comment:
```suggestion
    text-overflow: ellipsis;
    overflow: hidden;
    white-space: nowrap;
    padding-right: 2rem;
```
-webkit-box will only work on chrome, right? Any reason why we can't use text-overflow?

Reply:
This actually works in all modern browsers (since ~2018 release date) according to `caniuse.com`, just not old IE which is deprecated now anyway.
The `line-clamp` approach is preferred in this case because you don't have to calculate height based on line-height and so on, it just automatically does the calculation. Your suggested approach would be okay for 1 line clamping, but as soon as it gets to be more than that it becomes more complicated to work out than just changing a number